redbox/models/settings.py

```diff
@@ -44,6 +44,7 @@ class Settings(BaseSettings):
     partition_strategy: Literal["auto", "fast", "ocr_only", "hi_res"] = "fast"
 
     elastic: ElasticCloudSettings | ElasticLocalSettings = ElasticLocalSettings()
+    root_index: str = "redbox-data"
```

Comment:
Do we want this to be the same as what's in the integration .env file?

Reply:
nah, just some sensible default i think?